application/src/main/java/org/togetherjava/tjbot/config/FeatureBlacklist.java

```diff
@@ -30,6 +33,10 @@ public FeatureBlacklist(Set<T> featureIdentifierBlacklist) {
      * @return true if a feature is enabled, false otherwise
      */
     public boolean isEnabled(T featureId) {
-        return !featureIdentifierBlacklist.contains(featureId);
+        boolean isBlackListed = featureIdentifierBlacklist.contains(featureId);
+        if (isBlackListed) {
+            logger.info(String.format("%s is blacklisted 😥", featureId));
```

Comment:
* use the loggers own formatting api, not urs (otherwise the string is also computed if info is disabled) - `logger.info("{} is blacklisted", featureId)`
* id change the text slightly for better UX into `Feature {} is disabled.`

Reply:
👍